integration/it-integration/src/main/java/se/inera/intyg/rehabstod/integration/it/stub/IntygstjanstRestIntegrationServiceStub.java

```diff
@@ -140,6 +140,7 @@ public PopulateFiltersResponseDTO getPopulatedFiltersForActiveSickLeaves(Populat
             getDoctorsFromSickLeaves(sickLeaves),
             diagnosisChapters,
             sickLeaves.size(),
+            sickLeaves.isEmpty(),
```

Comment:
Shouldn't it be the opposite? If the list is empty we don't have any active sickleaves?